### DISCUSSION TITLE
Loading a profile always opens to OneDrive folder

### DISCUSSION THREAD

**Discussion by FabTheZen** [AntiMicroX/antimicrox#410] [Q&A] [answered]
Hi all, as per title, whenever I click on "Load", the file browser opens in the OneDrive folder instead of the last selected folder (or the program folder) as I'd expect. I'm opening a question instead of a bug/feature because I'm not sure if it's intended behavior, but legacy AntiMicro behaved as I described I would expect. Is this as it should be?

**Comment by pktiuk** [comment, marked as answer]
This is not expected, this seems to be a bug.

**Reply by FabTheZen** [reply]
Thanks! I'll open an issue then
